integrations/pgvector/src/haystack_integrations/document_stores/pgvector/filters.py

```diff
@@ -21,6 +21,19 @@
 NO_VALUE = "no_value"
 
 
+def _validate_filters(filters: Optional[Dict[str, Any]] = None):
```

Comment:
I think the errors that this method can raise are currently not covered by any tests. We could add test cases that trigger the two errors to increase coverage. It's not a must though.